LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2016 - 2021 Blue Brain Project/EPFL
+Copyright (c) 2016 - 2021-2022 Blue Brain Project/EPFL
```

Comment:
@pramodk @alex4200 I guess the automation didn't work well here?
Thanks!

Reply:
Will manually fix